kore/app/parser/Main.hs

```diff
@@ -137,36 +142,36 @@ mainVerify definition = flip runLoggerT Log.emptyLogger $ do
 {- | Validate a pattern relative to the provided module.
 
 If the module is not provided, no validation is performed.
-
 -}
-verifyPattern
-    :: Maybe (VerifiedModule Attribute.Symbol)
-    -- ^ Module containing definitions visible in the pattern.
-    -> ParsedPattern -- ^ Parsed pattern to check well-formedness
-    -> Main ()
+verifyPattern ::
+    -- | Module containing definitions visible in the pattern.
+    Maybe (VerifiedModule Attribute.Symbol) ->
+    -- | Parsed pattern to check well-formedness
+    ParsedPattern ->
+    Main ()
```

Comment:
This is a very unconventional way of formatting signatures. The usual way has its problems, but I'm not sure the solution is "format function signatures differently from every Haskell project on the planet".